shared/src/main/scala/com/github/vertical_blank/sqlformatter/scala/AbstractSqlFormatter.scala

```diff
@@ -1,5 +1,29 @@
 package com.github.vertical_blank.sqlformatter.scala
 
 trait AbstractSqlFormatter {
-  def format(sql: String): String
+  final def format(sql: String): String =
+    format(sql, SQLLanguage.StandardSQL, indent = "  ")
+
+  final def format(sql: String, indent: String): String =
+    format(sql, SQLLanguage.StandardSQL, indent = indent)
+
+  final def format(sql: String, params: Map[String, String]): String =
+    format(sql, params, SQLLanguage.StandardSQL, indent = "  ")
+
+  final def format(sql: String, params: Seq[String]): String =
+    format(sql, params, SQLLanguage.StandardSQL, indent = "  ")
+
+  def format(sql: String,
+             language: SQLLanguage,
+             indent: String): String
+
+  def format(sql: String,
+             params: Map[String, String],
```

Comment:
@takayahilton In Java implementation, named parameter allows Any types as value.
So, it's better that type of `params` is `Map[String, AnyVal]`.
Or considering `scala.math.BigInt` and `scala.math.BigDecimal` , `Map[String, Any]`.